integrationlink/show.go

```diff
@@ -0,0 +1,51 @@
+package integrationlink
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/olekukonko/tablewriter"
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/io"
+	"github.com/Scalingo/cli/utils"
+)
+
+func Show(app string) error {
+	c, err := config.ScalingoClient()
+	if err != nil {
+		return errgo.Notef(err, "fail to get Scalingo client")
+	}
+
+	repoLink, err := c.SCMRepoLinkShow(app)
+	if err != nil {
+		return errgo.Notef(err, "fail to get integration link for this app")
+	}
+	if repoLink == nil {
+		io.Statusf("Your app '%s' has no integration link.\n", app)
+		return nil
+	}
+
+	i, err := c.SCMIntegrationsShow(repoLink.AuthIntegrationUUID)
+	if err != nil {
+		return errgo.Notef(err, "fail to get integration information of this integration link")
+	}
+
+	t := tablewriter.NewWriter(os.Stdout)
+	t.SetHeader([]string{
```

Comment:
A table for a single element ?

Why not using something that is a bit more readable like:

```
App: myapp (58ab56456def)
Integration: gitlab-self-hosted (scm-asd4556-5645-564)

Repo: Scalingo/api
Branch: prod
Owner: johnsudaar

AutoDeploy: true

Review Apps: Enabled
   Destroy on close: 1h
   Destroy on Stale: 2h
```

Reply:
![Screenshot from 2019-08-30 12-44-05](https://user-images.githubusercontent.com/1567651/64015040-e81cb480-cb23-11e9-8f58-654d90a3c30d.png)

I used the same color as for the display of logs archive